frontend/test/metabase/scenarios/visualizations/scatter.cy.spec.js

```diff
@@ -0,0 +1,76 @@
+import { restore, visitQuestionAdhoc } from "__support__/cypress";
+import { SAMPLE_DATASET } from "__support__/cypress_sample_dataset";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATASET;
+
+const testQuery = {
+  database: 1,
+  query: {
+    "source-table": ORDERS_ID,
+    aggregation: [["count"], ["distinct", ["field-id", ORDERS.PRODUCT_ID]]],
+    breakout: [["datetime-field", ["field-id", ORDERS.CREATED_AT], "month"]],
+  },
+  type: "query",
+};
+
+describe("scenarios > visualizations > scatter", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+    cy.server();
+  });
+
+  it("should show correct labels in tooltip (metabase#15150)", () => {
+    cy.route("POST", "/api/dataset").as("dataset");
```

Comment:
if we have this in both tests, it's completely ok to extract it to top-most `beforeEach`.